apps/docs-snippets2/src/wallet/getBalance.test.ts

```diff
@@ -0,0 +1,10 @@
+import { main } from './getBalance.wrapped';
+
+/**
+ * @group node
+ * @group browser
+ */
+test('it works', async () => {
+  const { balances } = await main();
+  expect(balances).toBeTruthy();
+});
```

Comment:
How to use `launchTestNode` without polluting the [code snippet](https://github.com/FuelLabs/fuels-ts/pull/2651/files#diff-410c18e6bfa8e46676076161046c8a19c44c71c3736ea45b56578606eb0cd3a4) below?